### PR TITLE
 add data playground archetypes

### PR DESCRIPTION
**Pull request by masskoder**

### CODE REVIEW

File: archetypes/data-playground.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+api_key: "Add Api key Here"
+api_type: "Add Api type Here"
+sdks: " Add sdk list seperate with comma"
+limits: "Add limits here"
+data: " Add data here"
+account_required: " Add required account"
+
+---
+
+Add data products showcase
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:18]: Should this be commented out?

Comment (masskoder) [reply archetypes/data-playground.md:20]: it has been added to give instructions to the users who wants to add a new playground

File: archetypes/data-playground.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: "{{ replaceRE "^[-0-9]{10}-" "" .Name | replaceRE "-" " " | title }}"
+slug: "{{ replaceRE "^[-0-9]{10}-" "" .Name }}"
+subtitle: "add subtitle here" 
+draft: true
+image: img/data-playground/
+link: "add github link"
+description: 'Add Description Here'
+license: "Add License Here"
+producer: "Add Producer Here"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:10]: Is the subtitle the type? For example, `subtitle: Stock Market Data`? Should a link to the producer's website be included. I currently see a property for the GitHub link. 

Also, the description on the detail view is different from what's on the list page. Should a `summary` property be added for what will be displayed on the list page?

File: archetypes/data-playground.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: "{{ replaceRE "^[-0-9]{10}-" "" .Name | replaceRE "-" " " | title }}"
+slug: "{{ replaceRE "^[-0-9]{10}-" "" .Name }}"
+subtitle: "add subtitle here" 
+draft: true
+image: img/data-playground/
+link: "add github link"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:7]:
```suggestion
link: "Add GitHub Link Here"
```

File: archetypes/data-playground.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+description: 'Add Description Here'
+summary: " Add summary here"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:9]:
```suggestion
summary: " Add Summary Here"
```

File: archetypes/data-playground.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+license: "Add License Here"
+producer_name: "Add Producer name Here"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:11]:
```suggestion
producer_name: "Add Producer Name Here"
```

File: archetypes/data-playground.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+producer_link: "Add Producer Link"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:12]:
```suggestion
producer_link: "Add Producer Link Here"
```

File: archetypes/data-playground.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+api_key: "Add Api key Here"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:13]:
```suggestion
api_key: "Add API Key Here"
```

File: archetypes/data-playground.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+api_type: "Add Api type Here"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:14]:
```suggestion
api_type: "Add API Type Here"
```

File: archetypes/data-playground.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+sdks: " Add sdk list seperate with comma"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:15]:
```suggestion
sdks: " Add SDKs List Separated By Comma(s) Here"
```

File: archetypes/data-playground.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+limits: "Add limits here"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:16]:
```suggestion
limits: "Add Limits Here"
```

File: archetypes/data-playground.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+data: " Add data here"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:17]:
```suggestion
data: " Add Data Here"
```

File: archetypes/data-playground.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+account_required: "Add required account"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:18]:
```suggestion
account_required: "Add Required Account Here"
```

File: archetypes/data-playground.md [hunk 13]
```diff
@@ -0,0 +1,23 @@
+---
+title: "{{ replaceRE "^[-0-9]{10}-" "" .Name | replaceRE "-" " " | title }}"
+slug: "{{ replaceRE "^[-0-9]{10}-" "" .Name }}"
+subtitle: "add subtitle here"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:4]:
```suggestion
subtitle: "Add Subtitle Here" 
```

Comment (daniellemaxwell) [reply archetypes/data-playground.md:4]: I believe that this change and the one below will make the text a little more consistent.

**Review by daniellemaxwell — APPROVED** [review]
I appreciate you double checking the types. This is a good start toward implementing the Data Playground!

**Review by daniellemaxwell — APPROVED** [review]
I appreciate you double checking the types. This is a good start toward implementing the Data Playground!

File: archetypes/data-playground.md (hunk 3 above)
Comment (bbengfort) [reply archetypes/data-playground.md:7]: Would this be better?

```suggestion
link: "https://github.com/rotationalio/[insert-link-here]"
```

File: archetypes/data-playground.md (hunk 7 above)
Comment (bbengfort) [reply archetypes/data-playground.md:13]: This should probably be a `bool` - e.g. is api key required, true or false?

File: archetypes/data-playground.md (hunk 12 above)
Comment (bbengfort) [reply archetypes/data-playground.md:18]: This should also be a `bool` - is the account required true/false

File: archetypes/data-playground.md (hunk 1 above)
Comment (bbengfort) [reply archetypes/data-playground.md:22]: Don't worry about this for now; the users can add any content here, including a table.

File: archetypes/data-playground.md [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+api_key: "Add Api key Here"
+api_type: "Add Api type Here"
+sdks: " Add sdk list seperate with comma"
+limits: "Add limits here"
+data: " Add data here"
+account_required: "Add required account"
+
+---
+
+Add data products showcase
```
Comment (bbengfort) [inline archetypes/data-playground.md:22]:
```suggestion

<!-- Add content for data playground here, including a table for data products showcase if available ⟶

```

**Pull request merged**